bundles/net.tourbook/src/net/tourbook/ui/views/tourDataEditor/TourDataEditorView.java

```diff
@@ -588,7 +590,7 @@
    private ColumnManager            _swimSlice_ColumnManager;
    private SwimSlice_TourViewer     _swimSlice_TourViewer         = new SwimSlice_TourViewer();
 
-   private FormToolkit              _tk;
+   private FormToolkit              _formToolkit;
```

Comment:
I changed them because `tk` means nothing to me. Does it mean "Telekommunikation" ?  "[to come](https://en.wikipedia.org/wiki/To_come_(publishing))" ??? `_pc` does it mean computer like a PC ?? I know you've been using them for 10 years so probably it means "something" for you but not for me. But since you ask, I will revert.

Reply:
Reverted to `_tk`

Reply:
_tk ... toolkit
_pc ... pixel converter